### PR TITLE
Ensured we have access to transform.center when working with dilation in the Transform Widget 

### PR DESCRIPTION
**Pull request by SonicScrewdriver**
## Summary:
Team-LC has noticed recently that the skip rate on our transformer widget was quite a bit higher than it is normally. After further investigation, we discovered that the "homothety"/dilation tool is spitting out a large barrage of errors and is no longer functional. We suspect this is likely the main cause for the increase in reports. 

@jeanettehead and I looked into the history of the file and noticed that a key line of code was accidentally shifted to be a part of a parent comment as part of a [PR that removed all references to flow in Perseus](https://github.com/Khan/perseus/pull/457).

`center: self.dilationCircle.centerPoint.coord`

 It looks like a pesky little newline was accidentally (and likely automatically) removed, causing us to lose access to the required "center" property for our dilation tool. 

Since this widget is deprecated and only sees minimal use in our international content, the visibility of the issue was incredibly low. Thankfully the new Perseus Widget Analyzer by @pmcgill88 helped us catch this issue by providing greater insight into how our widgets are being actively used. 

Issue: LC-1384

## Test plan:
manual testing

### CODE REVIEW

File: packages/perseus/src/widgets/transformer.tsx [hunk 1]
```diff
@@ -2521,6 +2521,7 @@ class Transformer extends React.Component<Props> {
                     // NOTE(kevinb): It's hard to convince that this.dilationCircle
                     // will still be defined here.                    center: self.dilationCircle.centerPoint.coord,
```
Comment (jeanettehead) [inline packages/perseus/src/widgets/transformer.tsx:2522]: the code you need is actually here in this comment. the newline got accidentally deleted

File: packages/perseus/src/widgets/transformer.tsx [hunk 2]
```diff
@@ -2518,9 +2518,8 @@ class Transformer extends React.Component<Props> {
                 // @ts-expect-error - TS2345 - Argument of type '{ type: "dilation"; center: any; scale: number; }' is not assignable to parameter of type 'PerseusTransformerTransformation'.
                 self.doTransform({
                     type: "dilation",
-                    // NOTE(kevinb): It's hard to convince that this.dilationCircle
```
Comment (SonicScrewdriver) [inline packages/perseus/src/widgets/transformer.tsx:2521]: Realized we don't need this comment at all any more, as we've now removed all the flow overrides! No need to convince typescript. :)

File: packages/perseus/src/widgets/transformer.tsx [hunk 3]
```diff
@@ -2518,9 +2518,8 @@ class Transformer extends React.Component<Props> {
...
-                    // will still be defined here.                    center: self.dilationCircle.centerPoint.coord,
                     scale: newRadius / oldRadius,
+                    center: self.dilationCircle.centerPoint.coord,
```
Comment (jeremywiebe) [inline packages/perseus/src/widgets/transformer.tsx:2522]: This is a super small nit.   But could you move this to where it was above?

**Review by jeremywiebe — APPROVED** [review]
Nice catch!

**Pull request merged**